install-deps.sh

```diff
@@ -194,10 +212,11 @@ if [ x`uname`x = xFreeBSDx ]; then
     exit
 else
     source /etc/os-release
+    map_version_id_to_codename
```

Comment:
the codename is only used when preparing the `sources.list` for debian and its derivatives. so probably we could move this line into `debian|ubuntu|devuan)` branch?